From: Jiri Kosina <jkosina@suse.cz>
Date: Fri, 6 Mar 2015 16:16:56 +0100
Subject: kgr: x86: make build possible on x86_64 only
Patch-mainline: submitted for review
References: fate#313296

All the plumbing (especially in entry.S) is there only for x86_64, and
x86_32 is not supported at the moment (and probably never will be).

Make it impossible to build kgraft on x86_32.

js: move it to x86_64 Kconfig

Reported-by: fengguang.wu@intel.com
Signed-off-by: Jiri Kosina <jkosina@suse.cz>
Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 arch/x86/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/x86/Kconfig
+++ b/arch/x86/Kconfig
@@ -17,6 +17,7 @@ config X86_64
 	depends on 64BIT
 	select X86_DEV_DMA_OPS
 	select ARCH_USE_CMPXCHG_LOCKREF
+	select HAVE_KGRAFT
 
 ### Arch settings
 config X86
@@ -125,7 +126,6 @@ config X86
 	select RTC_LIB
 	select HAVE_DEBUG_STACKOVERFLOW
 	select ARCH_SUPPORTS_ATOMIC_RMW
-	select HAVE_KGRAFT
 
 config INSTRUCTION_DECODER
 	def_bool y
